### ISSUE TITLE
Other environmental datasets?

### ISSUE THREAD

**Issue by camfindlay** [data-govt-nz/odd17#1]
Feel free to add some of your favourites here too.

**Comment by genegrey** [comment]
I see tonnes on the data.govt.nz site - difficult to know what's missing without going through all of those (well over a thousand). On the other hand - wh0000! So many aesome envt datasets on data.govt.nz!

**Comment by camfindlay** [comment]
Yup, I've put a general link to the environmental section of data.govt.nz in the readme :)
